etc/ci/update_nightlies.sh

```diff
@@ -62,14 +62,15 @@ MASTER_COMMIT="$(git rev-parse HEAD)"
 
 git checkout -b gh-pages upstream/gh-pages || exit 1
 
-git rm -rf nightly || true
-rm -rf nightly || true
+OLD_NIGHTLIES="$(find nightly -type f -mtime +1)"
+git rm -rf $OLD_NIGHTLIES || true
```

Comment:
It's slightly nicer to only run this command if `$OLD_NIGHTLIES` is nonempty; you can wrap it in `if [ ! -z "$OLD_NIGHTLIES" ] ...`.  (`git rm -rf` fails on an empty list; `rm -rf` does not.  No idea why.)